CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+## Contributors guidelines
+ 
+### Getting started
+To start contributing to the Onfido Web SDK, clone this repository and run the following commands:
+ 
+- `npm install` to install the project dependencies
+- `npm run dev` to generate the build and run the dev server
+ 
+The SDK will be running on https://localhost:8080.
+ 
+Note: The SDK runs on `https`, which is needed to support the camera functionality.
+ 
+You can also run it using `docker` and `docker-compose` by running the following command: `docker-compose -f docker-compose.dev.yml`.
+ 
+### Testing
+The Web SDK has a large coverage of UI tests. To set up your environment, please refer to the [testing guidelines](./test/TESTING_GUIDELINES.md).
+Once your environment is ready, you can run UI tests by running `npm run build:test && npm run travis && npm run test:ui`.
+The Web SDK has partial coverage of unit tests that can be executed with `npm run test`.
+You can also run a type checker and linter by running `npm run check`.
+ 
+### Contributing
+When creating a new branch, contributors should use the following convention `{task-type}/{task-description}-{ticket-number}`.
+The most used task types are `feature`, `fix` or `improvement`. The ticket number is optional and is only expected from internal contributors.
+ 
+For more details, check out the [pull request checklist](./.github/PULL_REQUEST_TEMPLATE.md).
+ 
+In order to merge your Pull Request, you should get an approval from 2 Onfido team members. All the PR checks must pass.
+ 
+The current Pull Request checks are
+- Bundlesize - A tool that monitors the SDK size increase
+- Travis CI - Our continuous integration tool
+ 
+At build time, our CI runs the following tests:
+- UI
+- Unit
+- Type checker
+- Linter
+- Dependencies vulnerability detection
+ 
+All of the above must be green in order for the build to succeed.
```

Comment:
It might be better to just say "must have passed"? In case Github UI for this changes or just to account for colour blindness.